docs/dev/dev_environment.md

```diff
@@ -490,3 +490,52 @@ invoke generate-app-config-schema
 ```
 
 This command can only guess the schema, so it's up to the developer to manually update the schema as needed.
+
+## Using Bot Framework Emulator
+
+This option is limited to testing an Azure bot, but allows you to test the bots functionality without the need to create, publish, deploy the bot app into the MS Teams infrastructure.
+
+There are a few requirements to get this up and running, the steps are provided below to help a user get started.
+
+1. Follow the instructions on setting up the [Azure bot](../../admin/platforms/microsoft_teams/#azure).
+
+    !!! note
+        You can skip the `messaging endpoint` definition while testing in the Bot Emulator.
+
+    !!! note
+        You can also skip adding MS Teams to the channels list.
+
+    ![example-azure-bot](../images/light/azurebot.png#only-light)
+    ![example-azure-bot](../images/dark/azurebot.png#only-dark)
+
+2. Now that your Azure bot is created, and you've followed all the steps to capture the `Microsoft App ID` and the `Value` from your `Client Secret Key`. You can setup your local development environment. Please follow the standard steps outlined in the [Full Docker Development Environment](#full-docker-development-environment) section.
+
+    !!! warn
+        In order for this to work while running on the same host machine you'll need to add the `extra_hosts` option to the nautobot and worker docker compose services. Add the following to the end of your `invoke.yml`.
+
+    ```yaml
+        ---
+        nautobot_chatops:
+        ...
+        compose_files:
+            ...
+            - "docker-compose.bot-framework.yml"
+    ```
+
+3. Install Bot Framework Emulator. Based on the OS download and install from GitHub [releases](https://github.com/microsoft/BotFramework-Emulator/releases)
```

Comment:
Is this installing on the local machine? Do we need to link to the installation docs instead of the github releases page?

Reply:
Yes, this is installing to the local machine, not to the container. Their installation docs is just a link to the GitHub releases page.

Reply:
Maybe we could link to [this page](https://github.com/Microsoft/BotFramework-Emulator/blob/main/README.md#download) instead? That one gives more information on supported platforms and if the installation process or supported platforms change, they should hopefully update that.